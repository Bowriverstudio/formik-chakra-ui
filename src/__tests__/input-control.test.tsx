import { render } from '@testing-library/react';
import { Form, Formik } from 'formik';
import * as React from 'react';
import InputControl from '../input-control';

test('Input Renders Correctly', () => {
  const { asFragment } = render(
    <Formik onSubmit={() => {}} initialValues={{ test: 'Input' }}>
      <Form>
        <InputControl name="test" />
      </Form>
    </Formik>
  );

  expect(asFragment()).toMatchSnapshot();
});
