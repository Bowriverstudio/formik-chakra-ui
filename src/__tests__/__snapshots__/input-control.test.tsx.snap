// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Renders Correctly 1`] = `
<DocumentFragment>
  <form
    action="#"
  >
    <div
      class="chakra-form-control css-oujskm"
      role="group"
    >
      <input
        class="chakra-input css-1u8qly9"
        id="test"
        name="test"
        value="Input"
      />
    </div>
  </form>
</DocumentFragment>
`;
